lib/util/read_repo_ast_util.js

```diff
@@ -36,6 +36,7 @@
  */
 
 const assert   = require("chai").assert;
+const async     = require("generator-async");
```

Comment:
co converts a function that returns a generator into a function that returns a promise. I need to convert a function that returns a generator into a sync function.  But I guess I can do it another way.